docs/usage.md

```diff
@@ -138,85 +147,28 @@ Specify the path to a specific config file (this is a core Nextflow command). Se
 
 Whilst the default requirements set within the pipeline will hopefully work for most people and with most input data, you may find that you want to customise the compute resources that the pipeline requests. Each step in the pipeline has a default set of requirements for number of CPUs, memory and time. For most of the steps in the pipeline, if the job exits with any of the error codes specified [here](https://github.com/nf-core/rnaseq/blob/4c27ef5610c87db00c3c5a3eed10b1d161abf575/conf/base.config#L18) it will automatically be resubmitted with higher requests (2 x original, then 3 x original). If it still fails after the third attempt then the pipeline execution is stopped.
 
-For example, if the nf-core/rnaseq pipeline is failing after multiple re-submissions of the `STAR_ALIGN` process due to an exit code of `137` this would indicate that there is an out of memory issue:
+For example, if the nf-core/hicar pipeline is failing after multiple re-submissions of the `MAPS_CUT` process due to an exit code of `137` this would indicate that there is an out of memory issue:
 
 ```console
```

Comment:
I think it would be a good idea to leave this example as is, or replace it with a full error message from MAPS_CUT

Reply:
updated the documentation to full error message from MAPS_CUT.